### PR TITLE
Add Monbase Info and Logo

### PR DESCRIPTION
**Pull request by Monbase**
[Add Monbase Info and Logo](https://github.com/trustwallet/assets/pull/27811)
logo png
[info.json.json](https://github.com/trustwallet/assets/files/15057502/info.json.json)

### CODE REVIEW

**Comment by Monbase** [comment]
PR #27811
Already paid with this pr
Kindly reuse the fee in this pr instead

**Comment by Cryptocool1** [comment]
Hello, please send your audit here

**Comment by Monbase** [comment]
Hi,
This is our audit
https://skynet.certik.com/projects/monbase

Vào Th 2, 22 thg 4, 2024 vào lúc 16:31 CryptoA ***@***.***>
đã viết:

File: blockchains/smartchain/assets/0x551FAAB1027Cc50eFAeA5BED092E330475C3Cd99/info.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "MonbaseCoin",
+  "type": "BEP20",
+  "symbol": "MBC",
+  "decimals": 18,
+  "website": "https://monbase.com/",
+  "description": "",
```
Comment (Cryptocool1) [inline blockchains/smartchain/assets/0x551FAAB1027Cc50eFAeA5BED092E330475C3Cd99/info.json:7]: Please add your description

Comment (Monbase) [reply blockchains/smartchain/assets/0x551FAAB1027Cc50eFAeA5BED092E330475C3Cd99/info.json:7]: "description": "Monbase is a digital asset exchange offering hundreds of cryptocurrencies for buying, selling, and trading. Users can register, complete KYC, deposit funds, and manage their investments using an exchange wallet. Live market data is also provided for market analysis.",

**Comment by Monbase** [comment]
Hello Cryptocool1,
Is there anything wrong with our PR?

**Comment by Monbase** [comment]
Please remove the tag "launchpad" and update the token info from info.json for us please

**Comment by Monbase** [comment]
@Cryptocool1 Hi
With PR 27811 and PR 28028, we wanted to add the token INFO and LOGO. But in PR 28028, only the token LOGO got added , I don't know why.
Please add both the token INFO and LOGO for us.

**Comment by Monbase** [comment]
@cryptomanz1 Hi
With PR 27811 and PR 28028, we wanted to add the token INFO and LOGO. But in PR 28028, only the token LOGO got added , I don't know why.
Please add both the token INFO and LOGO for us.

**Comment by Monbase** [comment]
The token info from the info.json file, related to the token like price data, price chart, related links to the website , etc ...

**Comment by Cryptocool1** [comment]
Your token is already listed , you can see it inside the app

**Comment by Monbase** [comment]
For some reasons, I used many devices but only can see the token logo, not any other info about our token.
Do you have any advices?

**Comment by Monbase** [comment]
> Your token is already listed , you can see it inside the app

The word "listed", what do you mean by that? Cause from our side, we can only see our token logo, any data from the info.json file haven't been updated inside the app yet.

**Comment by Monbase** [comment]
Hello @Cryptocool1 , @cryptomanz1 
It's clearly said in the request that "Add Monbase Info and Logo", but right now, we are checking both in app and browser, only the Logo got added. The token Info is still missing.

Please guide us and update the token info for us

**Comment by Monbase** [comment]
image
image
"No price data available" and "An unknown error occurred"

**Pull request merged**